File: Armstrong.py
print("Armstrong number checker(within interval)")
print("**Here you can check Armstrong numbers available within your desired interval**")
no_range = list(map(int,input('Enter range').split()))

print("Armstrong numbers between",no_range[0],"&",no_range[1],"is: \n")
for num in range(no_range[0], no_range[1] + 1):
   # initialize sum
    sum = 0
    temp = num
    while temp > 0:
        digit = temp % 10
        sum += digit ** 3
        temp //= 10
    if sum == num:
        print(num)

File: armstrong.c
#include<stdio.h>  
int main()    
{    
  int n,r,sum=0,temp;    
  printf("enter the number=");    
  scanf("%d",&n);    
  temp=n;    
  while(n>0)    
  {    
    r=n%10;    
    sum=sum+(r*r*r);    
    n=n/10;    
  }    
  if(temp==sum)    
    printf("Given number is an armstrong  number ");    
  else    
    printf("Given number is not an armstrong number");    
  return 0;  
  }   

/*
Output:
enter the number=153
Given number is an armstrong number
*/
